apps/test-bundles/webpackUtils.js

```diff
@@ -73,6 +74,47 @@ function createWebpackConfig(entries) {
 // Files which should not be considered top-level entries.
 const TopLevelEntryFileExclusions = ['index.js', 'version.js', 'index.bundle.js'];
 
+function createEntries(packageName, isNorthstar) {
```

Comment:
I suggest to separate this to two functions: `createFluentFixtures()` and `createFluentNorthstarFixtures()`